errata_tool/erratum.py

```diff
@@ -722,13 +722,14 @@ def setFileInfo(self, file_info):
         self._processResponse(r)
 
     def removeBuilds(self, buildlist):
-        if type(buildlist) is not str and type(buildlist) is not list:
```

Comment:
This is the only non-trivial type()  to isinstance() conversion

from tests added in test_remove_builds.py I verified that isinstance(foo, (<type1>, <type2>)) does a is instance either type1 or type2, so this should be the clean refactor.